resources/clients/react/elwing/src/plugins/item-tracker/README.md

```diff
@@ -0,0 +1,104 @@
+# Work item tracker Elwing Plugin
+
+## Overview
+
+Shows how to use [React](https://reactjs.org/) to create a web page that connects to a REST service that lets you do the following:
+
+- Get a list of active or archived work items.
+- Mark active work items as archived.
+- Add new work items to the list of active items.
+- Send a report of work items to a specified email recipient.
+
+## Sample REST applications
+
+The web client is designed to send requests to one of the following sample applications.
+Each sample application shows you how to use an AWS SDK to store work items using AWS
+resources:
+
+- [Creating a React and Spring REST application that queries Amazon Aurora Serverless data](/javav2/usecases/Creating_Spring_RDS_%20Rest)
+- [Creating the Amazon Aurora Serverless backend using the AWS SDK for PHP](/php/cross_service/aurora_item_tracker)
+- [Tracking work items in an Aurora Serverless database with the SDK for Python](/python/cross_service/aurora_item_tracker)
+
+## Running the Item Tracker UI
+
+1.  Run Elwing by following the instructions in the [Elwing README](/resources/clients/react/elwing/README.md).
+1.  When Elwing starts, a web browser opens and browses to http://localhost:3000/.
+1.  Run the item tracker plugin by selecting **Item Tracker** in the left navigation bar.
+1.  This sends a request to the REST service to get any existing active items:
+    ```
+    GET http://localhost:8080/api/items?archived=false
+    ```
+1.  At first, the table is empty.
+
+    ![Work item tracker](images/item-tracker-start.png)
+
+1.  Select **Add item**, fill in the values, and select **Add** to add an item.
+
+    ![Add item](images/item-tracker-add-item.png)
+
+    This sends a POST request to the REST service with a JSON payload that contains the
+    work item.
+
+    ```
+    POST http://localhost:8080/api/items
+    {"name":"Me",
+     "guide":"Rust",
+     "description":"Show how to add an item",
+     "status":"In progress",
+     "archived":false}
+    ```
+
+1.  After you've added items, they're displayed in the table. You can archive an active
+    item by selecting the **Archive** button next to the item.
+
+    ![Work item tracker with items](images/item-tracker-all-items.png)
+
+    This sends a PUT request to the REST service, specifying the item ID and the
+    `archive` action.
+
+    ```
+    PUT http://localhost:8080/api/items/8db8aaa4-6f04-4467-bd60-EXAMPLEGUID:archive
+    ```
+
+1.  Select a filter in the dropdown list, such as **Archived**, to get and display
+    only items with the specified status.
```

Comment:
1.  To get and display items with a specific status, select a filter in the dropdown list, such as **Archived**.